### PR TITLE
Closes #720 | Add Sentiment-Annotated Taglish Product and Service Reviews dataloader

### LINKED ISSUE #720: Create dataset loader for Sentiment-Annotated Taglish Product and Service Reviews (SentiTaglish: Products and Services)
Dataloader name: `sentiment_taglist_product_review/sentiment_taglist_product_review.py`
DataCatalogue: http://seacrowd.github.io/seacrowd-catalogue/card.html?sentiment_taglist_product_review

|     Dataset| sentiment_taglist_product_review |
|-------------|---|
| Description | Sentiment-Annotated Taglish Product and Service Reviews (SentiTaglish: Products and Services) is a gold standard, sentiment-annotated corpus for the Tagalog-English language pair. The data set was created using publicly available online service and product reviews from Google Maps Reviews and Shopee Philippines. It contains 10,510 product and service reviews which were manually labeled by three human annotators according to four sentiment classes: Positive, Negative, Neutral, and Mixed. |
| Subsets | - |
| Languages | tgl, eng |
| Tasks | Sentiment Analysis, Text Classification |
| License | Creative Commons Attribution 4.0 (cc-by-4.0) |
| Homepage | https://huggingface.co/datasets/ccosme/SentiTaglishProductsAndServices |
| HF URL | https://huggingface.co/datasets/ccosme/SentiTaglishProductsAndServices |
| Paper URL | - |

### PR DESCRIPTION
**Pull request by ElijahGut**
Closes #720

### Checkbox
- [x] Confirm that this PR is linked to the dataset issue.
- [x] Create the dataloader script `seacrowd/sea_datasets/{my_dataset}/{my_dataset}.py` (please use only lowercase and underscore for dataset folder naming, as mentioned in dataset issue) and its `__init__.py` within `{my_dataset}` folder.
- [x] Provide values for the `_CITATION`, `_DATASETNAME`, `_DESCRIPTION`, `_HOMEPAGE`, `_LICENSE`, `_LOCAL`, `_URLs`, `_SUPPORTED_TASKS`, `_SOURCE_VERSION`, and `_SEACROWD_VERSION` variables.
- [x] Implement `_info()`, `_split_generators()` and `_generate_examples()` in dataloader script.
- [x] Make sure that the `BUILDER_CONFIGS` class attribute is a list with at least one `SEACrowdConfig` for the source schema and one for a seacrowd schema.
- [x] Confirm dataloader script works with `datasets.load_dataset` function.
- [x] Confirm that your dataloader script passes the test suite run with `python -m tests.test_seacrowd seacrowd/sea_datasets/<my_dataset>/<my_dataset>.py` or `python -m tests.test_seacrowd seacrowd/sea_datasets/<my_dataset>/<my_dataset>.py --subset_id {subset_name_without_source_or_seacrowd_suffix}`.
- [ ] If my dataset is local, I have provided an output of the unit-tests in the PR (please copy paste). This is OPTIONAL for public datasets, as we can test these without access to the data files.

### CODE REVIEW

File: seacrowd/sea_datasets/sentiment_taglish_product_review/sentiment_taglish_product_review.py
Comment (ljvmiranda921) [inline seacrowd/sea_datasets/sentiment_taglish_product_review/sentiment_taglish_product_review.py:45]: I double-checked #720 and the HF dataset and it seems like it should just be CC-by-4.0 🤔 

```suggestion
_LICENSE = Licenses.CC_BY_4_0.value
```

**Review by ljvmiranda921 — COMMENTED** [review]
Just one small edit but all good! Tested and the dataset worked 👍 
Let's do the change, wait for another reviewer's approval, then we can merge

**Comment by ElijahGut** [comment]
Hi LJ,

Thanks for the catch! I've updated the license now.

**Review by ljvmiranda921 — APPROVED** [review]

**Review by jensan-1 — APPROVED** [review]
Hello @ElijahGut 
Thank you for implementing the dataloader. Tested and LGTM.
Merging this (cc:@ljvmiranda921)

**Pull request merged**
